src/guides/v2.3/howdoi/checkout/checkout_payment_new_field.md

```diff
@@ -0,0 +1,322 @@
+---
+layout: tutorial
+group: how-do-i
+subgroup:
+title: Add a custom field in offline payment method
+contributor_name: Ziffity
+contributor_link: https://www.ziffity.com/
+subtitle: Customize Checkout
+menu_order: 3
+level3_subgroup: checkout-tutorial
+functional_areas:
+  - Checkout
+---
+
+The following steps are required to add a custom field to an offline payment method:
+
+1. Create a new module.
+1. Add a `db_schema.xml` file.
+1. Add a `requirejs` file to the module.
+1. Override the vendor files.
+1. Add an Observer.
+1. Verify that the module works.
+
+Let’s go through each step.
+
+## Step 1: Create a new module
+
+Create a new module named `Learning/CustomField` and register it.
+
+For more information about creating a new module, refer to [Create a Module](https://devdocs.magento.com/videos/fundamentals/create-a-new-module/)
+
+## Step 2 Add a `db_schema.xml` file
+
+Add the `paymentpocomment` column in the `quote_payment` and `sales_order_payment` table using the declarative schema method.
+
+Create the declarative schema in the following path `app/code/Learning/CustomField/etc/db_schema.xml`:
+
+{% collapsible Show code %}
+
+```xml
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="quote_payment" resource="checkout" engine="innodb" comment="Sales Flat Quote Payment">
+        <column xsi:type="text" name="paymentpocomment" nullable="true" comment="PO Comment"/>
+    </table>
+    <table name="sales_order_payment" resource="sales" engine="innodb" comment="Sales Flat Order Payment">
+        <column xsi:type="text" name="paymentpocomment" nullable="true" comment="PO Comment"/>
+    </table>
+</schema>
+```
+
+## Step 3: Add a requirejs file to the module
+
+Next, create the `requirejs-config.js` file:
+
+`app/code/Learning/CustomField/view/frontend/requirejs-config.js`
+
+```js
+ var config = {
+    map: {
+        '*': {
+          'Magento_OfflinePayments/js/view/payment/offline-payments':'Learning_CustomField/js/view/payment/offline-payments',
+        }
+    }
+}
+```
+
+{% endcollapsible %}
+
+## Step 4: Override the vendor files
+
```

Comment:
Provide a bulleted list of the files that need to overridden. Add a head3 section for each file.

Reply:
Modified. Also, added the head3 section.